### PR TITLE
Animals Table updated

### PR DESCRIPTION
**Pull request by ishimwepati**
Hi, 

below are the features I implemented :

1. I created a table named owners :
Screenshot 2023-09-29 at 22 02 26

2. I creayed a table named species : 
Screenshot 2023-09-29 at 22 02 52

Screenshot 2023-09-29 at 22 05 00

3. I Insert the following data into the owners table : 
Screenshot 2023-09-29 at 22 13 51

4. I insert the following data into the species table :
Screenshot 2023-09-29 at 22 17 03

5. I modify your inserted animals to include owner information (owner_id) 👍 
Screenshot 2023-09-29 at 23 08 26

6. I wrote queries (using JOIN) to answer the following questions:

- What animals belong to Melody Pond?
Screenshot 2023-09-29 at 23 15 41

- List of all animals that are Pokemon (their type is Pokemon).
Screenshot 2023-09-29 at 23 17 46

- List all owners and their animals, including those who don't own any animal.
Screenshot 2023-09-29 at 23 19 13

- How many animals are there per species?
Screenshot 2023-09-29 at 23 20 17

- List all Digimon owned by Jennifer Orwell.
Screenshot 2023-09-29 at 23 21 30

-  List all animals owned by Dean Winchester that haven't tried to escape.
Screenshot 2023-09-29 at 23 22 39

-Who owns the most animals?
Screenshot 2023-09-29 at 23 23 34

Your suggestions are most welcome !

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -11,6 +11,46 @@ CREATE TABLE animals (
...
+CREATE TABLE animals_new (
+    id serial PRIMARY KEY,
+    name text,
+    date_of_birth date,
+    escape_attempts integer,
+    neutered boolean,
+    weight_kg decimal,
+    species_id integer REFERENCES species(id),
+    owner_id integer REFERENCES owners(id)
+);
```
Comment (abdulhamiid) [inline schema.sql]: Please add the query used to modify the animal table, following the steps in the [project requirements ](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md)🤗

image


This helps to ensure that you have actually run the queries with success. 💪🤓

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @ishimwepati 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Use primary key & foreign key mechanisms for joining tables.
- ✔️ Query multiple tables.
- ✔️ Prepare complex queries that answer analytical questions.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (ishimwepati) [reply schema.sql]: Thanks a lot, Below are all the steps I used :
Screenshot 2023-09-30 at 02 16 50

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @ishimwepati 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Use primary key & foreign key mechanisms for joining tables.
- ✔️ Query multiple tables.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**